mjsip-phone/src/main/java/org/mjsip/phone/gui/StringList.java

```diff
@@ -68,7 +68,7 @@ protected void saveFile(File file) {
 			writeTo(new FileWriter(file));
 		}
 		catch (IOException e) {
-			LOG.error("Failed writing file \""+file+"\"", e);
+			LOG.error("Failed writing file ({})", file, e);
```

Comment:
Is this really equivalent? The pattern has only one placeholder, and there are two arguments. Is this equivalent to the signature with message and exception?

Reply:
Not 100%. 

The old entry produces `Failed writing file "filename"` the new one produces `Failed writing the file (filename)`

Reply:
And yes, both print an additional stack trace.

Reply:
That's good to know, thank you!